<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Digitalfilters_NIDAQ_widget</class>
 <widget class="QWidget" name="Digitalfilters_NIDAQ_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>757</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>537</width>
    <height>407</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>537</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Digital Filters</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(64, 64, 64);
}

QTabWidget::pane { 
   border: 1px solid rgb(166, 166, 166);
}

QTabBar::tab {
  	background-color: rgb(77, 77, 77);
 }

 QTabBar::tab:selected {
  	background-color: rgb(140, 140, 140);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:selected:hover {
  	background-color: rgb(140, 140, 140);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:hover {
  	background-color: rgb(109, 109, 109);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:middle {
	border-right: 1px dashed rgb(166, 166, 166);
	border-left: 1px dashed rgb(166, 166, 166);
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:last {
	border-top-right-radius: 10px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }

 QTabBar::tab:first {
	border-top-left-radius: 10px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 12px;
	padding-right: 12px;
 }


QComboBox QAbstractItemView {
    background-color: rgb(77, 77, 77);
}

QComboBox QAbstractItemView::item:focus{
    background-color: rgb(140, 140, 140);
}

QDoubleSpinBox{
	background-color: rgb(77, 77, 77);
	
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}

QDoubleSpinBox::up-button{
    image: url(:/imgs/drop_up_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::up-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::up-button:hover{
	background-color: rgb(0, 50, 0);
}

QDoubleSpinBox::down-button{
    image: url(:/imgs/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::down-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::down-button:hover{
	background-color: rgb(0, 50, 0);
}

QWidget#centralwidget{
	background-color: rgb(64, 64, 64);
}

QWidget{
	color: rgb(255, 255, 255);
}

QComboBox{
	background-color: rgb(77, 77, 77);
	
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}


QComboBox::drop-down{
	image: url(:/imgs/imgs/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 2px solid rgb(127, 167, 127);
	border-left: 2px solid rgb(127, 167, 127);

	border-bottom: 2px solid rgb(0, 0, 0);
	border-right: 2px solid rgb(0, 0, 0);
}

QComboBox::drop-down:hover{
	background-color: rgb(0, 50, 0);
}

QComboBox::drop-down:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QPushButton{
	background-color: rgb(0, 79, 0);

	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);

	
	font: 12pt &quot;Helvetica&quot;;
	text-align:center;
}

QWidget{
	font: 12pt &quot;Helvetica&quot;;
}

QPushButton:hover{
	background-color: rgb(0, 50, 0);
}

QPushButton:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QLineEdit{
	background-color: rgb(77, 77, 77);
	border-top: 1.5px solid rgb(46, 46, 46);
	border-left: 1.5px solid rgb(46, 46, 46);

	border-bottom: 1.5px solid rgb(166, 166, 166);
	border-right: 1.5px solid rgb(166, 166, 166);
}

QRadioButton::indicator{
	border-radius: 6px;
	border-top: 1.5px solid rgb(0, 0, 0);
	border-left: 1.5px solid rgb(0, 0, 0);

	border-bottom: 1.5px solid rgb(160, 160, 160);
	border-right: 1.5px solid rgb(160, 160, 160);
}

QRadioButton::indicator::checked{
	background-color: white;
}

QRadioButton::indicator::unchecked:hover{
	background-color: #9F9F9F;
}

QRadioButton::indicator::pressed{
	border: 1.5px solid #505050
}

QPushButton#reload_devices{
	image: url(:/imgs/imgs/reload.png);
	width: 11px;
	background-color: rgb(0, 79, 0);

	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);

	
	font: 12pt &quot;Helvetica&quot;;
	text-align:center;
}

QPushButton#reload_devices:hover{
	background-color: rgb(0, 50, 0);
}

QPushButton#reload_devices:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::up-button{
    image: url(:/imgs/imgs/drop_up_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::up-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::up-button:hover{
	background-color: rgb(0, 50, 0);
}

QDoubleSpinBox::down-button{
    image: url(:/imgs/imgs/drop_down_arrow.png);
	width: 11px;

	background-color: rgb(0, 79, 0);
	border-top: 1.5px solid rgb(127, 167, 127);
	border-left: 1.5px solid rgb(127, 167, 127);

	border-bottom: 1.5px solid rgb(0, 0, 0);
	border-right: 1.5px solid rgb(0, 0, 0);
}

QDoubleSpinBox::down-button:pressed{
	border: 2px solid rgb(255, 255, 255);
}

QDoubleSpinBox::down-button:hover{
	background-color: rgb(0, 50, 0);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="left_side" native="true">
     <property name="minimumSize">
      <size>
       <width>483</width>
       <height>108</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>108</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="2" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="yes_fr">
           <property name="text">
            <string>Yes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">ratio_fr</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="no_fr">
           <property name="text">
            <string>No</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">ratio_fr</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="5">
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Plot frequency response?</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Select your filter:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QComboBox" name="type_filter">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>22</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>FIR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IIR</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>560</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QPushButton" name="save_button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QWidget" name="iir_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>519</width>
       <height>259</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>259</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Order:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_11" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <widget class="QComboBox" name="designbox_iir">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Chebyshev Type I</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Chebyshev Type II</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Butterworth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Elliptic</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_13" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_16">
         <item row="0" column="0">
          <widget class="QLineEdit" name="rp">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_9" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0" alignment="Qt::AlignRight">
          <widget class="QLineEdit" name="order_iir">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_14" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_17">
         <item row="0" column="0">
          <widget class="QLineEdit" name="rs">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="6">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Filter Design:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_10" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QLineEdit" name="cutoff_iir">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Stopband Attenuation (dB):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Cutoff:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Passband Ripple (dB):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_12" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <widget class="QComboBox" name="typebox_iir">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>lowpass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>highpass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bandpass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bandstop</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="fir_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>519</width>
       <height>289</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>519</width>
       <height>289</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="3" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="0" column="0">
          <widget class="QComboBox" name="type_box">
           <property name="minimumSize">
            <size>
             <width>145</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>145</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>lowpass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>highpass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bandpass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bandstop</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QGridLayout" name="gridLayout_5"/>
       </widget>
      </item>
      <item row="4" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="cutoff_widget" native="true">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLineEdit" name="cutoff_fir">
           <property name="text">
            <string>0.1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" alignment="Qt::AlignRight">
          <widget class="QLineEdit" name="order_fir">
           <property name="font">
            <font>
             <family>Helvetica</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="6">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="1" column="0" alignment="Qt::AlignRight">
          <widget class="QComboBox" name="design_box">
           <property name="minimumSize">
            <size>
             <width>145</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Hamming</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hann</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bartlett-Hann</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blackman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gauss</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Kaiser</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="2" alignment="Qt::AlignRight">
       <widget class="QWidget" name="fc_widget" native="true">
        <layout class="QGridLayout" name="gridLayout_18">
         <item row="0" column="0" alignment="Qt::AlignRight">
          <widget class="QLineEdit" name="fc1_line">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>0.1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="fc2_line">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>0.2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="bd_widget" native="true">
        <layout class="QGridLayout" name="gridLayout_19">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Cutoff (Hz):</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="cof_widget" native="true">
        <layout class="QGridLayout" name="gridLayout_20">
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Cutoff (Hz):</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="widget_16" native="true">
        <layout class="QGridLayout" name="gridLayout_21">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="widget_17" native="true">
        <layout class="QGridLayout" name="gridLayout_22">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Order:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignLeft">
       <widget class="QWidget" name="widget_18" native="true">
        <layout class="QGridLayout" name="gridLayout_23">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Filter Design:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources_1.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="ratio_fr"/>
 </buttongroups>
</ui>
